src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background: url("/images/star.svg") no-repeat center;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+  }
+
+  &--1 .stars__star:nth-child(-n+1),
+  &--2 .stars__star:nth-child(-n+2),
+  &--3 .stars__star:nth-child(-n+3),
+  &--4 .stars__star:nth-child(-n+4),
+  &--5 .stars__star:nth-child(-n+5) {
+    background-image: url("/images/star-active.svg");
+  }
```

Comment:
you can use a @for loop for this and also `&--1 .stars__star:nth-child(-n+1)` can be coded like `&--1 &:nth-child(-n+1)`

Reply:
Okay thanx done.